samples/ipc/ipc_service/sample.yaml

```diff
@@ -15,34 +15,34 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
     extra_args:
       SB_CONF_FILE=sysbuild/nrf5340dk_nrf5340_cpunet.conf
-      ipc_service_OVERLAY_CONFIG=rpmsg.conf
-      remote_OVERLAY_CONFIG=rpmsg.conf
+      EXTRA_CONF_FILE=rpmsg.conf
+      remote_EXTRA_CONF_FILE=rpmsg.conf
 
   sample.ipc.ipc_service.nrf5340dk_rpmsg_cpuapp_sending:
     platform_allow:
       - nrf5340dk_nrf5340_cpuapp
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
     extra_configs:
-      - ipc_service_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=1
+      - CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=1
       - remote_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=200000000
```

Comment:
move all `remote_CONFIG*` from `extra_configs` to `extra_args` to pass Twister

Reply:
Ok. Done. Will it be fixed later in twister?

Reply:
I don't know. In docs is written `Extra configuration options to be merged with a master prj.conf `, maybe should be extended to support also sysbuild or some extra clarification could be added. For now using `extra_args` is enough